### ISSUE TITLE
fix: [aria-selected attribute not updated when removing selected attribute] in [fluent-option?]

### ISSUE THREAD

**Issue by JohnLemireMSFT** [microsoft/fast#5923]
# 🐛 Bug Report

I am using an observable array and an observable currentItem to populate a select and set the option's selected prop
I have noticed when I do this if I change the currentItem the selected attr is properly removed from the previously selected option and added to currently selected item but the derivative aria-selected attr is set true on the new one and left as true on the old one causing  both items to render with the vertical accent bar on the start side.

sorry I don't have the code on this laptop

## 🤔 Expected Behavior

The aria-selected attr state should be updated when the selected attr is removed

## 😯 Current Behavior
multiple options are rendered as selected

## 🌍 Your Environment

* OS & Device: [e.g. MacOS, iOS, Windows, Linux] on [iPhone 7, PC]
* Browser [e.g. Microsoft Edge, Google Chrome, Apple Safari, Mozilla FireFox]
* Version [e.g. 1.8.0]

**Comment by chrisdholt** [comment]
Thanks @JohnLemireMSFT for this issue - I think some kind of repro steps will be helpful here just to get down to the nuance of how things are getting setup. I did do a quick pass to just confirm that this is working for the option itself - when setting the property the attribute is reflected as expected and changes to `"false"` when removing it. 

If you can provide a small repro, perhaps even using something like https://stackblitz.com/, that will help us identify where the specific issue your seeing is coming from.

**Comment by janechu** [comment]
Closing due to deprecation of Foundation per #6955.
